### PR TITLE
Look for SSH config correctly in multi cluster environments

### PR DESCRIPTION
**Pull request by dippynark**
**What this PR does / why we need it**: In multi-cluster environments, tarmak k8s clusters should look for vault cluster ssh credentials in the hub's folder rather than look in the cluster's folder

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/tarmak/cluster/cluster.go [hunk 1]
```diff
@@ -338,7 +338,14 @@ func (c *Cluster) ConfigPath() string {
 	return filepath.Join(c.Environment().Tarmak().ConfigPath(), c.ClusterName())
 }
 
+func (c *Cluster) HubName() string {
+	return fmt.Sprintf("%s-%s", c.environment.Name(), clusterv1alpha1.ClusterTypeHub)
+}
+
```
Comment (simonswine) [inline pkg/tarmak/cluster/cluster.go]: This should ideally live in pkg/environment and then called through the environment interface

**Comment by simonswine** [comment]
@dippynark I still need to merge this or is this already included in #100?

**Comment by dippynark** [comment]
@simonswine I have merged these changes into https://github.com/jetstack/tarmak/pull/100 so this no longer needs merging

**Comment by simonswine** [comment]
@dippynark thanks for the clarification

**Pull request closed without merging**